.github/workflows/rspec.yml

```diff
@@ -26,6 +26,17 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
+    # https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix
+    strategy:
+      fail-fast: false
+      matrix:
+        # Set N number of parallel jobs you want to run tests on.
+        # Use higher number if you have slow tests to split them on more parallel jobs.
+        # Remember to update ci_node_index below to 0..N-1
+        ci_node_total: [8]
```

Comment:
- Without parellelization rspec took [16m 51s](https://github.com/rubyforgood/diaper/actions/runs/423664255).
- With parellelization across 10 nodes, it took [6m 55s](https://github.com/rubyforgood/diaper/actions/runs/423858228)
- With parellelization across 8 nodes, it took [6m 55s](https://github.com/rubyforgood/diaper/actions/runs/423789455)
- With parellelization across 6 nodes, it took [7m 14s](https://github.com/rubyforgood/diaper/actions/runs/423842123)
- With parellelization across 8 nodes for the second time, it took [6m 39s](https://github.com/rubyforgood/diaper/actions/runs/423967381)